src/spritecache_internal.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file spritecache_internal.h Internal functions to cache sprites in memory. */
+
+#ifndef SPRITECACHE_INTERNAL_H
+#define SPRITECACHE_INTERNAL_H
+
+#include "stdafx.h"
+
+#include "core/math_func.hpp"
+#include "gfx_type.h"
+#include "spriteloader/spriteloader.hpp"
+
+#include "table/sprites.h"
+
+/* These declarations are internal to spritecache but need to be exposed for unit-tests. */
+
+struct SpriteCache {
+	void *ptr;
+	size_t file_pos;
+	SpriteFile *file;    ///< The file the sprite in this entry can be found in.
```

Comment:
Often you remove the extra spaces when changing/moving the lines. Was that intended to happen here too?

Reply:
Didn't really think of it, just copy & paste...